### DISCUSSION TITLE
How to use PointsMaterial in TresJs?

### DISCUSSION THREAD

**Discussion by schachmatthias** [Tresjs/tres#891] [General]
I thought with the naming convention its just `<TresMeshPointsMaterial  />` but that doesnt work. Does it exist under a different name? Any help is much appreciated.

https://threejs.org/docs/?q=points#api/en/materials/PointsMaterial

**Comment by andretchen0** [comment]
Hi @schachmatthias ,

Try `<TresPointsMaterial />`.

We don't provide support here, but if you need further help, [you can ask over on the Discord](https://discord.com/invite/WYj3pjpPUK). It's usually faster.

**Comment by andretchen0** [comment]
Closing: support request
